README.md

```diff
@@ -46,6 +46,18 @@ Example usage:
 python icone_translator.py -i '../sample files/icone data/incidents_extended.xml' 
 ```
 
+### Execution for COtrip translator
+
+#### Step 1: Run the translator script (from Work_Zone/translator/source code/Folder)
+
+```
+python cotrip_translator.py -i inputfile.xml -o outputfile.geojson
+```
+Example usage:
+```
+python cotrip_translator.py -i '../sample files/icone data/incidents_extended.xml' 
```

Comment:
Running this command produces the following error:
![image](https://user-images.githubusercontent.com/2932159/113169850-3a8fec80-9203-11eb-8302-0b637454cc37.png)